koku/api/common/tests_pagination.py

```diff
@@ -93,6 +93,21 @@ def test_get_previous_link_value(self, mock_super):
         link = paginator.get_previous_link()
         self.assertEqual(link, expected)
 
+    def get_paginated_response_schema(self):
+        """Test the returned pagination schema."""
+        data = {
+            'test_data': 'test'
+        }
+
+        meta = {
+            'additional_meta': 123
+        }
+        pagination_schema = \
```

Comment:
Very minor style nitpick: the koku project has its line length set to 120. So, you don't need this hard line break.

See tox.ini:
```
max-line-length = 120
```